spec/components/check_answers_summary_list_card_component_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+RSpec.describe CheckAnswersSummaryListCardComponent, type: :component do
+  context "when given a set of questions" do
+    let(:user) { FactoryBot.build(:user) }
+    let(:case_log) { FactoryBot.build(:case_log, :completed) }
+    let(:subsection_id) { "household_characteristics" }
+    let(:subsection) { case_log.form.get_subsection(subsection_id) }
+    let(:questions) { subsection.applicable_questions(case_log) }
+
+    it "renders a summary list card for the answers to those questions" do
```

Comment:
I feel like I might be able write both of those assertions in one test using the details known question 🤔

Reply:
I've tried to write these suggested tests as unit tests for the component 🙂

Reply:
I think here that maybe leads to testing the implementation rather than the outcome given that those are effectively "private" methods that aren't expected to be used anywhere outside this component I think that might be making the idea of the "unit" too small.